### ISSUE TITLE
Add convenience functions for creating kraken, diamond, bwa databases

### ISSUE THREAD

**Issue by tomkinsc** [broadinstitute/viral-ngs#519]
We should have convenience functions for building kraken, diamond, and metagenomic bwa databases, accessible as CLI subcommands of `metagenomics.py`, similar to the [`taxon_filter.py`](https://viral-ngs.readthedocs.io/en/latest/taxon_filter.html) commands `lastal_build_db`, `blastn_build_db`, and `bmtagger_build_db`.

The `tools/kraken.py` file provides a `build()` function as a member of the `Kraken` class, but this should be exposed as a script command. The argument parser should specify the inputs needed for a custom DB, and handle any pre-processing, such as creation of `library/` and `taxonomy/`.

**Comment by tomkinsc** [comment]
@yesimon: Would you have time to add these functions? They would support later additions to automate db rebuild with new accessions. I like how easy [VirMet](https://github.com/ozagordi/VirMet/blob/master/virmet/common.py) makes it to rebuild their metagenomics DB with the latest full set of viral genomes.
https://virmet.readthedocs.io/en/latest/updating/

**Comment by yesimon** [comment]
Yup - these are already there in part for the integration tests. Just need to add CLI
